packages/variable-editor/src/components/variables/data/variable.ts

```diff
@@ -1,16 +1,19 @@
+import type { ValidationMessages } from '@axonivy/variable-editor-protocol';
 import type { TreeNode, TreeNodeFactory, TreeNodeUpdate, TreeNodeUpdates } from '../../../utils/tree/types';
 import type { Metadata } from './metadata';
 
 export interface Variable extends TreeNode<Variable> {
   description: string;
   metadata: Metadata;
+  validations: ValidationMessages;
```

Comment:
why not make this optional, so you don't have to add it to all your test cases?
Technically this would also be fine or not? it has validations (then it is an array) or it has nothing